tests/test_testing.py

```diff
@@ -60,7 +159,7 @@ def receive(self, text_data=None, bytes_data=None):
         pass
 
 
-@pytest.mark.asyncio
+@pytest.mark.django_db
```

Comment:
It will be a sequencing thing - it definitely doesn't belong here, so it may be that your cleanup code is bad. Try running the tests in different orders.